cats/shared/src/main/scala/caseapp/cats/IOCaseApp.scala

```diff
@@ -65,11 +65,20 @@ abstract class IOCaseApp[T](implicit val parser0: Parser[T], val messages: Help[
   def stopAtFirstUnrecognized: Boolean =
     false
 
+  /**
+    * Whether to ignore unrecognized arguments.
+    *
+    * That is, if there are unrecognized arguments, the parsing still succeeds.
+    * The unparsed arguments are put in the `args` argument of `run`.
+    */
+  def ignoreUnrecognized: Boolean =
```

Comment:
I believe this parameter should be taken into account in `def parser` above, and `Parser.withHelp` too. The call to `parser.withHelp` below returns a `Parser` wrapping `parser`, and the `ignoreUnrecognized` parameter can only gets through thanks to this, IIRC.

The tests of case-app are missing tests for `CaseApp` and `IOCaseApp` themselves. Passing those parameters this way is required to have them work.

Reply:
Thanks for the quick feedback, I'll address this soon.